src/state/slices/marketDataSlice/selectors.ts

```diff
@@ -45,24 +45,14 @@ const selectAssetId = (_state: ReduxState, assetId: AssetId) => assetId
 export const selectMarketDataById = createCachedSelector(
   selectMarketData,
   selectAssetId,
-  selectFiatMarketData,
-  selectSelectedCurrency,
-  (cryptoMarketData, assetId, fiatMarketData, selectedCurrency): MarketData => {
+  (cryptoMarketData, assetId): MarketData => {
     const defaultMarketData: MarketData = {
       price: '0',
       marketCap: '0',
       volume: '0',
       changePercent24Hr: 0,
     }
-    const assetMarketData = cryptoMarketData[assetId] ?? defaultMarketData
-    const fiatPrice = bnOrZero(fiatMarketData[selectedCurrency]?.price ?? 1) // fallback to USD
-    if (fiatPrice.eq(1)) return assetMarketData // don't unnecessarily compute price history for USD
-    return {
-      ...assetMarketData,
-      price: bnOrZero(assetMarketData?.price ?? 0)
-        .times(fiatPrice)
-        .toString(),
-    }
```

Comment:
@shapeshift/operations FYI we will need to fully regression test fiat (USD or user-selected currencies) market data across the app, this removes changes introduced in https://github.com/shapeshift/web/pull/1859/

Reply:
@gomesalexandre a little note here, this functionality still exists in `selectMarketData`, since these `cryptoMarketData` here is using the mentioned selector, the rates are already in selected fiat currency. this avoids the `rate^2` problem.